README.md

```diff
@@ -1,23 +1,40 @@
 # project-template
 Project template based on gulp, jade and postcss.
 ## Install
-```
-git clone https://github.com/yakhinvadim/project-template
-cd project-template
-npm install
-```
-then you can run tasks, described below
+
+Action | Command
+:--- | :---
+clone repository | `git clone https://github.com/yakhinvadim/project-template your-project-name`
+enter project folder | `cd project-template`
+install gulp globally | `npm install -g gulp`
+install dependencies | `npm install`
+
+after installing, you can run tasks, described below
 ## Tasks
 
+### Primary tasks
+
+Command | Task
+:--- | :---
+`gulp build` | build /dist from /src
+`gulp watch` | build, then watch for changes in /src and automatically run secondary tasks (below) to process these changes
+`gulp` | build, run localhost:4000 and watch for changes
+
+### Secondary tasks
+
 Command | Task
 :--- | :---
-`gulp build` | build 'dist' folder from 'src' folder
-`gulp watch` | watch for changes in source folders and automatically run tasks to process these changes
-`gulp html` | compile .jade files to .html files
-`gulp css` | concatenate .css files, process css with postcss-processors, create source-map for result css
-`gulp js` | copy .js files from src to dist folder without changes
+`gulp html` | compile .jade to .html
+`gulp css` | concatenate .css, process with postcss processors and create source-map
+`gulp js` | copy .js to /dist without changes
 `gulp img` | optimize images
 `gulp icons` | build svg-sprite from separate svg-icons
-`gulp ftp` | upload dist folder to ftp (don't forget to provide credentials and upload address in gulpfile.js)
-`gulp clean` | delete dist folder |
-`static-server dist` | run localhost:9080
+`gulp temp` | copy temporarily files to /dist without changes
```

Comment:
temp folder will contain mostly images, that won't be used in production.
also there will be production related css and js